src/graph.rs

```diff
@@ -470,7 +471,68 @@ fn is_false(v: &bool) -> bool {
   !v
 }
 
-#[derive(Debug, Default, Clone, Serialize, PartialEq, Eq)]
+fn serialize_import_assertions<S>(
+  assertions: &ImportAssertions,
+  serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+  S: Serializer,
+{
+  if let Some(map) = assertions.maybe_static() {
+    map.serialize(serializer)
```

Comment:
Serialization here is unstable because the underlying structure is a `HashMap`. Probably import assertions should be stored as an `IndexMap` so it's stable.